broadbean/blueprint.py

```diff
@@ -504,24 +530,26 @@ def changeDuration(self, name, dur, replaceeverywhere=False):
 
         # Validation
         if name not in self._namelist:
-            raise ValueError('No segment of that name in blueprint.'
-                             ' Contains segments: {}'.format(self._namelist))
+            raise ValueError(
+                "No segment of that name in blueprint."
+                " Contains segments: {}".format(self._namelist)
+            )
 
         for name in replacelist:
             position = self._namelist.index(name)
 
             if dur <= 0:
-                raise ValueError('Duration must be strictly greater '
-                                 'than zero.')
+                raise ValueError("Duration must be strictly greater " "than zero.")
```

Comment:
There are a bunch of places where black has made this formatting 

```suggestion
                raise ValueError("Duration must be strictly greater than zero.")
```

Reply:
thanks for pointing it out, I'll get this fixed.